.github/actions/build/action.yaml

```diff
@@ -5,7 +5,7 @@ runs:
   steps:
     - name: Set up Cloud SDK
       if: startsWith(${{matrix.target_platform}}, 'android')
-      uses: isarkis/setup-gcloud@40dce7857b354839efac498d3632050f568090b6 # v1.1.1
+      uses: google-github-actions/setup-gcloud@v2
```

Comment:
Weren't we required to always use a git hash here as immutable release ? Or is that not required for "official" repos ?

Reply:
We only need hashes for actions that aren't hosted by Google or GitHub.